testing/unittests/jlibtests.cpp

```diff
@@ -296,12 +292,12 @@ class JlibTraceTest : public CppUnit::TestFixture
 
         //retrieve serverSpan context with the intent to propagate it to a remote child span
         {
-            Owned<IProperties> retrievedSpanCtxAttributes = createProperties();
-            bool getSpanCtxSuccess = serverSpan->getSpanContext(retrievedSpanCtxAttributes.get(), true);
+            Owned<IProperties> retrievedClientHeaders = createProperties();
+            serverSpan->getClientHeaders(retrievedClientHeaders.get());
 
-            CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected getSpanContext failure detected", true, getSpanCtxSuccess);
+            //CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected getClientHeaders failure detected", true, retrievedClientHeaders->length()>0);
```

Comment:
Should this test be uncommented again?

Reply:
Changed it to ensure at least the parentspan was provided.